pom.xml

```diff
@@ -2439,7 +2439,7 @@
       <properties>
         <alt-hadoop>mapr</alt-hadoop>
         <rat.excludeSubprojects>true</rat.excludeSubprojects>
-        <hive.version>2.1.1-mapr-1710</hive.version>
+        <hive.version>2.3.3-mapr</hive.version>
         <hbase.version>1.1.1-mapr-1602-m7-5.2.0</hbase.version>
         <hadoop.version>2.7.0-mapr-1707</hadoop.version>
         <zookeeper.version>3.4.5-mapr-1710</zookeeper.version>
```

Comment:
Update Zookeeper version to 3.4.11-mapr-1808?

Reply:
There is a dedicated PR for this: https://github.com/apache/drill/pull/1398.

Reply:
Thanks!